src/NHSD.BuyingCatalogue.Organisations.Api/Models/Policy.cs

```diff
@@ -3,6 +3,7 @@
     internal static class Policy
     {
         public const string CanAccessOrganisation = "CanAccessOrganisation";
```

Comment:
Just to confirm, did we agree these should be plural?

Reply:
We totally did. Gonna fix that :]